WORKSPACE

```diff
@@ -0,0 +1,58 @@
+workspace(
+    name = "builder",
+    managed_directories = {
+        "@npm": ["node_modules"],
+    },
+)
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# Fetch rules_nodejs so we can install our npm dependencies
+http_archive(
+    name = "build_bazel_rules_nodejs",
+    sha256 = "1134ec9b7baee008f1d54f0483049a97e53a57cd3913ec9d6db625549c98395a",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/3.4.0/rules_nodejs-3.4.0.tar.gz"],
+)
+
+# Check the rules_nodejs version and download npm dependencies
+# Note: bazel (version 2 and after) will check the .bazelversion file so we don't need to
+# assert on that.
+load("@build_bazel_rules_nodejs//:index.bzl", "check_rules_nodejs_version", "node_repositories", "npm_install")
+
+check_rules_nodejs_version(minimum_version_string = "2.2.0")
+
+# Setup the Node.js toolchain
+node_repositories(
+    node_version = "15.0.0",
```

Comment:
We're currently on node v12 / v10 in production, and the majority of packages use the < v14 / npm6 lockfile.

Reply:
latest LTE is 14 so maybe we go w/ that

Reply:
👍